dto/organization_dto.go

```diff
@@ -3,16 +3,14 @@ package dto
 import "github.com/checkmarble/marble-backend/models"
 
 type APIOrganization struct {
-	Id                         string `json:"id"`
-	Name                       string `json:"name"`
-	ExportScheduledExecutionS3 string `json:"export_scheduled_execution_s3"`
```

Comment:
Be mindfull, if you merge this, `export_scheduled_execution_s3` won't be available in org schema anymore (with get request).
This will break the BO due to this :
```ts
const OrganizationSchema = yup.object({
  id: yup.string().required(),
  name: yup.string().defined(),
  export_scheduled_execution_s3: yup.string().defined(),
});
```

Reply:
done in https://github.com/checkmarble/marble-backoffice/pull/26